README.md

```diff
@@ -32,6 +32,7 @@ the app is wireframe only.
 ##  Author 1 - aku
 
 - GitHub: aakbarkhan.
+- [aakbarkhan](https://github.com/aakbarkhan)
```

Comment:
- [ ] It's good that you added the link to your Github account 👍 however as the previous CR already mentioned, to make your repo look more professional your "Author" section should look like the [template ](https://github.com/microverseinc/readme-template) provided by Microverse. see below example as well

## Authors

👤 **Author1**

- GitHub: [@githubhandle](https://github.com/githubhandle)
- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)

Reply:
Thank you.